WORKSPACE

```diff
@@ -1253,6 +1253,17 @@ load("@rules_pkg//:deps.bzl", "rules_pkg_dependencies")
 
 rules_pkg_dependencies()
 
+# System Call test dependencies.
+http_archive(
```

Comment:
Could you add a comment stating that this is before grpc so it takes precedence over their absl version?

Reply:
I wasn't going to ask, but if you're added a comment anyways, maybe note that this commit is lts_2021_03_24?

Reply:
a6e5a17 is this ok?